docs/Usage/Modules/EsminiAdapter.md

```diff
@@ -17,4 +18,14 @@ Load an OpenSCENARIO-file called `MyScenario.xosc`.
 
 - `open_scenario_file: "MyScenario.xosc"`
 
+## Test origin
 
+The test origin is extracted from the OpenDRIVE file of the scenario. To change the test origin to a different location, change the "geoReference" tag in the OpenDrive file header. 
+
+*Example*
+```xml
+<OpenDRIVE>
+  <header ... >
+    <geoReference><![CDATA[+proj=tmerc +lat_0=57.77752905043481 +lon_0=12.7814573051027 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +geoidgrids=egm96_15.gtx +vunits=m +no_defs ]]></geoReference>
```

Comment:
An optional geoidgrids file location could be specified in params.yaml for example. When I try to run dev right now it will not work until i place a egm96_15.gtx file within my workspace (My scenario has a projstring that requires a geoidgrid file).

Reply:
@Robert108